### PR TITLE
Defined notes for BSI SYS.1.6.A17

### PR DESCRIPTION
**Pull request by lichtblaugue**
#### Description:

Added notes and controls for BSI SYS.1.6 A17

#### Rationale:

As we have multiple customers asking for a BSI profile to be included in the compliance-operator, we are contributing a profile. To provide a better review process, the individual controle are implemented as separate PRs.

### CODE REVIEW

File: applications/openshift/general/sandboxed_containers_operator_configured/rule.yml
Comment (sluetze) [inline applications/openshift/general/sandboxed_containers_operator_configured/rule.yml:1]: please do not add the same content, which is already existing in other PRs / commits. just rebase on that branch or wait until the other branch is merged

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: applications/openshift/general/sandboxed_containers_operator_configured/tests/ocp4/e2e-remediation.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# for the kataconfig
+echo "waiting for sandboxed-containers-operator pods to be ready"
+oc wait -n openshift-sandboxed-containers-operator --for=condition=ContainersReady=true \
+--timeout=300s pods -l control-plane=controller-manager
+
+echo "configuring kataconfig"
+oc apply -f ${ROOT_DIR}/ocp-resources/e2e/sandboxed-containers-instance.yaml --server-side=true
+
+echo "check, that the mcp was updated"
+oc wait --for=condition=Updated --timeout=3600s machineconfigpool/kata-oc
```
[the remark this answers is not in the document]
Comment (sluetze) [reply applications/openshift/general/sandboxed_containers_operator_configured/tests/ocp4/e2e-remediation.sh:24]: Hi @yuumasato ,

this PR should not include https://github.com/ComplianceAsCode/content/pull/12530/files/8e36b6428e1e65122ec3aa8b1cac0284c06c1ad1..14ae93f598e984653248f96215f068f6f746d2e4

as this is done in https://github.com/ComplianceAsCode/content/pull/12494

I will help @lichtblaugue to remove this commits from the PR and consider your comment in the pull/12494

**Comment by lichtblaugue** [comment]
Closing in favor of 
https://github.com/ComplianceAsCode/content/pull/12663

**Pull request closed without merging**